habitat-hitl/habitat_hitl/environment/controllers/gui_controller.py

```diff
@@ -153,66 +152,66 @@ def act(self, obs, env):
         # if isinstance(arm_ctrlr, ArmEEAction):
         #     EE_FACTOR = 0.5
         #     # End effector control
-        #     if gui_input.get_key_down(KeyNS.D):
+        #     if gui_input.get_key_down(KeyCode.D):
         #         arm_action[1] -= EE_FACTOR
-        #     elif gui_input.get_key_down(KeyNS.A):
+        #     elif gui_input.get_key_down(KeyCode.A):
```

Comment:
is there a reason this commented chunk is kept around?

Reply:
That was reference code, but I doubt that we'll be using any of that. Removed.